plugins/score-card/.prettierignore

```diff
@@ -0,0 +1 @@
+src/api/*-client.ts
```

Comment:
Should we really be doing this? If Prettier has suggestions that should be fixed it might be better to fix them

Reply:
good catch! it was a left over from our sync with some other plugin where these -client.ts were autogenerated. I will remove it.

Reply:
removed. I believe this is it so I will also resolve the conversation so its not messy here. Thanks for the comment!